CMakeLists.txt

```diff
@@ -85,22 +88,20 @@ set_target_properties (qrack PROPERTIES
 
 # Install common headers
 install (FILES
+    include/common/oclengine.hpp
     include/common/complex16simd.hpp
     include/common/complex16x2simd.hpp
     include/common/complex8x2simd.hpp
-    include/common/oclengine.hpp
+    include/common/parallel_for.hpp
```

Comment:
Unnecessary ordering change?

Reply:
I can revert it, but parallel_for.hpp needs to go in.